prow/config.yaml

```diff
@@ -51,31 +51,11 @@ github_reporter:
   - postsubmit
 
 blocked_branches: &blocked_branches
-  release-0.1: &blocking_merge
+  release-*: &blocking_merge
```

Comment:
a) [Tide blockers](https://github.com/kubernetes/test-infra/tree/master/prow/tide/blockers) is a better way to achieve this effect. Ask @cjwagner about it. The [blockade plugin](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/blockade) is something else to consider.

b) I suspect admins simply press the merge button? The [override plugin](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/override/override.go) is probably a better alternative to use here if you're not already aware of it.

c) yaml anchors tend to make it harder to understand these config files, which limits the number of people willing to maintain them. Best to avoid them and inline the values each time. Use a script to generate the file when/if this becomes onerous.

Reply:
As mentioned previously, most of our repos don't use tide and we're moving istio/api and istio/istio off of tide as well. So I don't believe the plugins you suggest  would work for us in that context.

As for the yaml anchors, I think it substantially increases readability and maintainability, so I'd like to keep 'em as-is. For most of our repos so far, very few humans have been dealing with branch protection stuff (i.e. me), so I'm not worried about a constraint on people not being familiar with the syntax.